gridstatus/ercot_api/ercot_api.py

```diff
@@ -1293,6 +1327,50 @@ def get_cop_adjustment_period_snapshot_60_day(
 
         return Ercot()._process_cop_adjustment_period_snapshot_60_day_data(data)
 
+    @support_date_range(frequency=None)
+    def get_system_load_charging_4_seconds(
+        self,
+        date: str | pd.Timestamp | tuple[pd.Timestamp, pd.Timestamp],
+        end: str | pd.Timestamp | tuple[pd.Timestamp, pd.Timestamp] | None = None,
+        verbose: bool = False,
+    ) -> pd.DataFrame:
+        if date == "latest":
+            return self.get_system_load_charging_4_seconds(
+                "today",
+                verbose=verbose,
+            )
+
+        api_params = {"AGCExecTimeFrom": date, "AGCExecTimeTo": end}
+
+        data = self.hit_ercot_api(
+            endpoint=ESR_ENDPOINT,
+            api=APITypeEnum.ESR_API,
+            **api_params,
+        )
```

Comment:
I haven't got the historical bulk download method working yet for this dataset, but, since this dataset only has data starting yesterday, I think that's fine for now.